vshn/website
# chore: update dependencies

**Describe what changes this pull request brings**
This pull request updates all dependencies, including `gatsby-plugin-sitemap`, which should fix the bug related to generation sitemap. 
Check sitemap [here](https://dependencies-update--vshn.netlify.app/sitemap/sitemap-index.xml)


**Describe any additional changes**


**Steps to test**
1. Open [the page](https://dependencies-update--vshn.netlify.app/sitemap/sitemap-0.xml)
2. Confirm that everything looks and works as expected


**Screenshots**


**Checklist**



**TODO**


**References**
- Closes the task [Fix sitemap.xml generation](https://www.notion.so/pixelpoint/Fix-sitemap-xml-generation-bbe0f5b1a6944023a50559ed0810ef3d)
- [gatsby-plugin-sitemap](https://www.gatsbyjs.com/plugins/gatsby-plugin-sitemap/)
